packages/web-twig/src/Resources/components/Tabs/TabLink.twig

```diff
@@ -12,7 +12,6 @@
 {# Attributes #}
 {%- set _ariaControlsAttr = _target ? 'aria-controls="' ~ _target | escape('html_attr') ~ '"' : null -%}
 {%- set _dataTargetAttr = _target ? 'data-target="#' ~ _target | escape('html_attr') ~ '"' : null -%}
```

Comment:
I know it is outside the scope of this issue, but shouldn't we filter this attribute from the main props?

Reply:
Good question. Now when we have `data-toggle` opt-in, it makes sense to remove also the "translation" of `target` to `data-target`.

Reply:
I will do not do this right now because the other aria attribute is dependent on this.